repos/exawind/packages/nalu-wind/package.py

```diff
@@ -49,6 +49,7 @@ class NaluWind(CmakeExtension, bNaluWind, ROCmPackage):
     depends_on("hypre+umpire", when="+umpire")
     depends_on("trilinos gotype=long")
     depends_on("openfast@develop+netcdf+cxx", when="+fsi")
+    patch("fortran.patch", when="+fsi %apple-clang")
```

Comment:
Technically this should be `+openfast` or `+fsi`.  
I think since we merged fsi stuff into `openfast@develop` we can change these variants relationship though.

Reply:
I think openfast needs special attention in its own PR.